src/fotmob.ts

```
@@ -145,6 +148,22 @@ export default class Fotmob {
     );
   }
 
+  async getLTC<T = LTC>(matchid: number, teams: string[]) {
+    const params = new URLSearchParams({
+        ltcUrl: `data.fotmob.com/webcl/ltc/gsm/${matchid}_en.json.gz`,
+        teams: JSON.stringify(teams)
+    });
+
+    const url = `${this.ltcUrl}?${params.toString()}`;
+    console.log(url);
+
+    return await this.safeTypeCastFetch<T>(
+        url,
+        (data) => JSON.parse(data) as T,
```

isn't it silently hides changes in the api with this type enforcment?